### PR TITLE
Feature/change trls singleton to translate method

### PR DESCRIPTION
**Pull request by rubeniskov**
https://github.com/k14v/i18njs/issues/27

### CODE REVIEW

**Comment by rubeniskov** [comment]
Available to merge

File: package.json [hunk 1]
```diff
@@ -69,25 +72,25 @@
   "dependencies": {
     "@k14v/printf-tokenize": "0.0.5",
     "@k14v/rexpmat": "0.0.3",
-    "mem": "5.1.1",
-    "printf": "0.5.1"
+    "mem": "^6.0.1",
```
Comment (Kelvur) [inline package.json:75]: I prefer do dependency updates in a separate PR

**Review by Kelvur — APPROVED** [review]
LGTM, but it's necessary update the readme documentation about `trls`; Algo, I made a commit replacing `trls.__` by `translate` in the documentation or comments

**Comment by Kelvur** [comment]
@rubeniskov I made the last commit about docs, It's prepared to merge
